tdrs-backend/tdpservice/settings/common.py

```diff
@@ -368,6 +370,32 @@ class Common(Configuration):
 
     ENABLE_DEVELOPER_GROUP = True
 
+    # XMS CONFIG
+    '''
+    "issuer":"https://int.xms.hhs.gov/",
+    "authorization_endpoint":"https://int.xms.hhs.gov/ABaaS/oidc/authorize",
+    "token_endpoint":"https://int.xms.hhs.gov/ABaaS/oidc/token",
+    "userinfo_endpoint":"https://int.xms.hhs.gov/ABaaS/oidc/userinfo",
+    "jwks_uri":"https://int.xms.hhs.gov/ABaaS/oidc/certificates"
+    '''
+    XMS_ACR_VALUES = os.getenv(
+        'ACR_VALUES',
+        'http://idmanagement.gov/ns/assurance/ial/1'
+    )
+    XMS_AUTHORIZATION_ENDPOINT = 'https://int.xms.hhs.gov/ABaaS/oidc/authorize'
+
+    XMS_CLIENT_ASSERTION_TYPE = ''
+    XMS_CLIENT_ID = 'U2Xzpq6_ZA_5M~Z'
+    XMS_ISSUER = "https://int.xms.hhs.gov/"
+    XMS_JWKS_ENDPOINT = "https://int.xms.hhs.gov/ABaaS/oidc/certificates"
+    # JWT_KEY must be set, there is no functional default.
+    XMS_JWT_KEY = 'n5i8ZxW.I8N68iobatUBR~ClLkc7t5q2~.oZXkx.N7.L-jvJuo'
```

Comment:
this seems like it should be an environment variable?

Reply:
Correct, will do the change